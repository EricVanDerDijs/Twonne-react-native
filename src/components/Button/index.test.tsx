import React from 'react';
import { Button } from './index';
import shallowRenderer from 'react-test-renderer/shallow';

const shallow = shallowRenderer.createRenderer();

function mockHandler(): void {
  // tslint:disable-next-line
  console.debug('test text')
}

it('Renders Button', () => {
  shallow.render((
    <Button
      text='exppected text'
      onPress={mockHandler}
    />
  ));
});
