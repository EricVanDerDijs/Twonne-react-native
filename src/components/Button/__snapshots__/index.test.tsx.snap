// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders NativeButton with our custom styles 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#FF9E54",
      "borderRadius": 10,
      "height": 50,
    }
  }
>
  <View
    style={null}
  >
    <Text
      style={
        Object {
          "color": "white",
        }
      }
    >
      exppected text
    </Text>
  </View>
</View>
`;
